integration/spark2/pom.xml

```diff
@@ -105,6 +105,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
```

Comment:
Is this required in this PR?

Reply:
I test this issue with CarbonThrfitServer and beeline in the local,  it throw the exception, so I fix it in this PR too.small change.

Reply:
@xubo245 This change is already there in #2925 .  We can discuss on that PR on this. Please remove the changes from this PR. 

@jackylk Let us discuss whether this is actually required. Please refer the PR #2925 and give inputs

Reply:
Please handle it fast，no this dependency will affect the carbonThriftServer